### DISCUSSION TITLE
Is it possible to collapse a tree item?

### DISCUSSION THREAD

**Discussion by caleb-at-pieces** [microsoft/vscode-discussions#1096] [Extension Development QnA] [answered]
I see that there is a nice api to expand a tree item such as:

```typescript
treeview.reveal(treeItem, {expand: true});
```

Is there a good way to collapse a tree item? I tried this same method but with `expand: false` however this doesn't seem work.

**Comment by tamuratak** [comment, marked as answer]
Seems not supported. Tracked at microsoft/vscode/issues/92176.

**Reply by burkeholland** [reply]
Marking as answer for now. Please feel free to re-open if you feel this is an error.
